generator/config/expressions.php

```diff
@@ -0,0 +1,163 @@
+<?php
+
+declare(strict_types=1);
+
+// Target namespace for the generated files, allows to use ::class notation without use statements
+
+namespace MongoDB\Builder\Expression;
+
+use MongoDB\BSON;
+use MongoDB\Builder\Pipeline;
+use MongoDB\Builder\Type;
+use MongoDB\CodeGenerator\Definition\PhpObject;
+use MongoDB\Model\BSONArray;
+use stdClass;
+
+use function ucfirst;
+
+$bsonTypes = [
+    // BSON types
+    // @see https://www.mongodb.com/docs/manual/reference/bson-types/
+    // Ignore deprecated types and min/max keys which are not actual types
+    'double' => ['int', BSON\Int64::class, 'float'],
+    'string' => ['string'],
+    'object' => ['array', stdClass::class, BSON\Document::class, BSON\Serializable::class],
+    'array' => ['array', BSONArray::class, BSON\PackedArray::class],
+    'binData' => ['string', BSON\Binary::class],
+    'objectId' => [BSON\ObjectId::class],
+    'bool' => ['bool'],
+    'date' => [BSON\UTCDateTime::class],
+    'null' => ['null'],
+    'regex' => [BSON\Regex::class],
+    'javascript' => ['string', BSON\Javascript::class],
```

Comment:
I wasn't aware that aggregation framework accepted a string alongside a Javascript object, but I assume this pertains to [$function](https://www.mongodb.com/docs/manual/reference/operator/aggregation/function/#mongodb-expression-exp.-function).